packages/clay-core/src/tree-view/TreeViewItem.tsx

```diff
@@ -340,14 +341,20 @@ export function TreeViewItemStack({
 
 	const hasChildren = nestedChildren && nestedChildren.length > 0;
 
+	const cssClasses = ['component-expander'];
+
+	Array.isArray(expanderButtonClasses)
+		? expanderButtonClasses.forEach((c) => cssClasses.push(c))
+		: cssClasses.push(expanderButtonClasses!);
+
 	return (
 		<Layout.ContentRow {...otherProps}>
 			{expandable && (
 				<Layout.ContentCol>
 					<Button
 						aria-controls={`${item.key}`}
 						aria-expanded={expandedKeys.has(item.key)}
-						className={classNames('component-expander', {
+						className={classNames(cssClasses, {
```

Comment:
You could have just added it that way wouldn't be a problem either.

```
classNames('component-expander', expanderButtonClasses, {
```

The property doesn't have to be an array, because it has the same effect as `'class other-class'`. It would have the same effect we do for the other components, we could also call it `expanderClassName` to follow the naming of other components.

Reply:
> You could have just added it that way wouldn't be a problem either.

Sorry I don't understand. 

> The property doesn't have to be an array, because it has the same effect as 'class other-class'. 

Yes, I know, if you want to revert this feel free, the thing is, I needed to get this done and merged - and you were on PTO. 

>  we could also call it expanderClassName to follow the naming of other components.

That's why I did in the first iteration, but I think it's better to be precise, these classes are going to be applied to the "Button" that wraps the "expander"  so `expanderButtonClasses` doesn't seem to be that "wrong" - also don't get me wrong but there is no much "consistency" with prop names in the whole repository.  I'll re submit a change since no one in DXP is using this yet.